src/Symfony/Component/Console/Application.php

```diff
@@ -1169,4 +1169,27 @@ private function splitStringByWidth($string, $width)
 
         return $lines;
     }
+
+    /**
+     * Returns all namespaces of the command name.
+     *
+     * @param string $name The full name of the command
+     *
+     * @return array The namespaces of the command
+     */
+    private function extractAllNamespaces($name)
+    {
+        $parts = explode(':', $name, -1);
```

Comment:
I have added a comment and squashed.

@carlescliment `extractNamespace` was extracting the full namespace of a command (`doctrine:database:create` -> `doctrine:database`) and `extractAllNamespaces` is extracting all the namespaces of a command, not just the full one (`doctrine` and `doctrine:database`). Do you have a better idea for the method name?